resources/qml/delegates/Placeholder.qml

```diff
@@ -0,0 +1,7 @@
+import ".."
+
+MatrixText {
```

Comment:
Recommend the display of this be tied to a User Setting.  Not everyone wants to see unsupported events on the timeline.

Reply:
Yeah, that's what I mentioned in the ToDo. I actually need to hide those events in the model, as we also don't want to have the sections appear, when there is no content in between. That could be easily achieved by just having two list, one of all event ids in order and one of only the shown event ids, but I didn't have the time yet, as I rather see all events first. I will implement a toggle for that before 0.7.0 stable though ^-^